### PR TITLE
Add bish

### PR DESCRIPTION
**Pull request by victorgama**
This adds `bish`.
From bish's [repository](https://github.com/tdenniston/bish):
```
Bish is a language that compiles to Bash. It's designed to give shell scripting a more comfortable and modern feel.
```

### CODE REVIEW

File: Library/Formula/bish.rb [hunk 1]
```diff
@@ -0,0 +1,27 @@
+class Bish < Formula
+  homepage "https://github.com/tdenniston/bish"
+  url "https://github.com/tdenniston/bish/archive/v0.1.tar.gz"
+  sha256 "796d1efcbc9e8c7ea604881860aa8d857aad089c4eeb766283c21c210687942b"
+
+  def install
+    stdlib_path = share/"stdlib.bish"
+    mkdir share
+    mv "lib/stdlib.bish", stdlib_path
```
Comment (DomT4) [inline Library/Formula/bish.rb]: You can replace this and the two lines above with ` share.install "lib/stdlib.bish" `.

Comment (victorgama) [inline Library/Formula/bish.rb]: @DomT4 Done! Didn't know there was a `.install` method. Sorry!

Comment (DomT4) [inline Library/Formula/bish.rb]: Thanks! Yeah - It essentially does the same thing, but it looks less messy in formulae which is always a :+1:.

Comment (victorgama) [inline Library/Formula/bish.rb]: :sunglasses:

**Comment by chdiza** [comment]
Isn't this a bit "too-new" for Homebrew's policies?  It's not even two months old, and it says right on the README that "you would probably be crazy to use Bish in any sort of production environment until it stabilizes a bit more."  There is one tagged release, 0.1.

**Comment by DomT4** [comment]
Potentially. It's not a self-submission, and it is popular in terms of Github attention. Homebrew has permitted new/non-production-ready things in that bracket, despite being new. If it's rejected here, could always place it in ` devel-only ` for now.

**Comment by victorgama** [comment]
I do not know any of the homebrew's policies, but I do think it would be interesting to have this on its index, despite bish's development state. I mean, brew would offer an easier way to install it on one's system, even because bish's makefile does not link it to `/usr/local/bin` or any other system path, leaving this task to whoever is installing it. To be honest, when I first saw it on GitHub, my first attempt to install it was through brew.

But again, this is just my opinion.

**Comment by mikemcquaid** [comment]
Yeh, passing on it for that reason. `homebrew-devel-only` may make better sense for now.

**Comment by victorgama** [comment]
@mikemcquaid How should I proceed? Open this same PR on Homebrew/homebrew-devel-only?

**Comment by DomT4** [comment]
I'm happy enough to merge a PR for this there, if Mike has no objections.

**Comment by victorgama** [comment]
Will open a PR there, and then you guys can decide. (:

-- 
Victor Gama
http://vito.io
+55 11 953 804 895

On April 22, 2015 at 3:13:14 PM, Dominyk Tiller (notifications@github.com) wrote:

I'm happy enough to merge a PR for this there, if Mike has no objections.

**Comment by mikemcquaid** [comment]
No objections to merging it there.

**Comment by victorgama** [comment]
Perfect! Already opened a PR there.
Thank you guys! :beer:

**Pull request closed without merging**